lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -390,6 +390,10 @@ probe_cb(void *cb_ctx, const struct spdk_nvme_probe_info *probe_info,
 		return false;
 	}
 
+	if (ctx->controllers_remaining > 0) {
```

Comment:
Let me confirm. Can we configure iscsi.conf as follows? If yes, the controllers_remaining change is necessary to prevent creating bdevs for 4 NVMe devices. If no, controllers_remaining can be deleted from nvme_probe_ctx structure.

[Nvme]
  BDF 0000:02:00.0
  BDF 0000:03:00.0
  BDF 0000:04:00.0
  BDF 0000:05:00.0
  NumControllers 2